src/server/world/terrain_generation.cpp

```diff
@@ -60,7 +61,8 @@ float getNoiseAt(const glm::vec2 &blockPosition, const glm::vec2 &chunkPosition,
         float x = voxelX * frequency / options.smoothness;
         float y = voxelZ * frequency / options.smoothness;
 
-        float noise = glm::simplex(glm::vec3{x, y, seed});
+        float *seed_float = reinterpret_cast<float *>(&seed);
```

Comment:
This is undefined behavior, I suggest using `std::bit_cast`
see https://en.cppreference.com/w/cpp/numeric/bit_cast

Reply:
Nevermind, std::bit_cast is c++20
You can use `std::memcpy` instead.